### ISSUE TITLE
Initial

### ISSUE THREAD

**Issue by maka89** [maka89/SA-MLP#1]
-Comments
-Loss function for classification
-Make method for calculating gradient(batch backprop)
